src/Tribe/Admin/Organizer_Chooser_Meta_Box.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * Class Tribe__Events__Admin__Organizer_Chooser_Meta_Box
+ *
+ * Handles the Organizer section inside the Events meta box
+ */
+class Tribe__Events__Admin__Organizer_Chooser_Meta_Box {
+	/**
+	 * @var WP_Post
+	 */
+	protected $event;
+
+	/**
+	 * @var Tribe__Events__Main
+	 */
+	protected $tribe;
+
+	public function __construct( $event = null ) {
+		$this->tribe = Tribe__Events__Main::instance();
+		$this->get_event( $event );
+	}
+
+	/**
+	 * Work with the specifed event object or else use a placeholder if we are in
+	 * the middle of creating a new event.
+	 *
+	 * @param null $event
+	 */
+	protected function get_event( $event = null ) {
+		global $post;
+
+		if ( $event === null ) {
+			$this->event = $post;
+		} elseif ( is_a( $event, 'WP_Post' ) ) {
```

Comment:
`is_a()` was briefly deprecated in PHP 5.2, thus throws deprecation notices for that version. Switching this to `$event instanceof WP_Post` should resolve that.